src/placeholders/index.d.ts

```diff
@@ -11,22 +11,22 @@ interface RectShapeProps {
   className?: string,
   style?: {}
 }
-export declare class RectShape extends Component<RectShapeProps, void> { }
+export declare class RectShape extends Component<RectShapeProps, any> { }
```

Comment:
The best strategy is probably to remove that type parameter altogether

```typescript
Component<RectShapeProps>
```

should be ok